cli/src/main/java/rules/CheckTagsAreDefinedRule.java

```diff
@@ -0,0 +1,23 @@
+package rules;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.openapitools.openapistylevalidator.ErrorAggregator;
+import org.openapitools.openapistylevalidator.custom.OpenAPIRule;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+
+/**
+ * This rule will check whether tags are defined in the rule section
+ */
+public class CheckTagsAreDefinedRule extends OpenAPIRule {
+
```

Comment:
Another thing to think is that some of the rules/validation have a parameter. I think we will need a `OpenAPIParameterRule` or something like that for them.

Reply:
Can you provide any sample where rules need parameter? If we need any rules that needs parameter, we can create some context object to wrap all the required parameters and pass it as parameter to `OpenAPIRule.execute()` method.

Reply:
Well, maybe we don't have a clear definition in the command line between a rule and a parameter. For example, this could be a rule that is activated (true/false): `validateNaming`

And there are a number of parameters that are used by this rule, for example `pathNamingConvention` takes one of the following enum values: `CamelCase`, `HyphenUpperCase`, `HyphenCase`, `UnderscoreCase`, `UnderscoreUpperCase`, `AnyCase`